google/cloud/grpc_options.h

```diff
@@ -81,6 +81,17 @@ struct GrpcNumChannelsOption {
  * values in different options as gRPC will use the first value set for some
  * channel arguments, and the last value set for others.
  *
+ * @par Example
+ * To configure gRPC's maximum metadata size to 32MiB (which can be useful
+ * to avoid `RESOURCE_EXHAUSTED` errors when exporting gRPC metrics),
+ * you can use:
+ * @code
+ *   google::cloud::Options options;
+ *   grpc::ChannelArguments args;
+ *   args.SetInt(GRPC_ARG_MAX_METADATA_SIZE, 32 * 1024 * 1024);
+ *   options.set<google::cloud::GrpcChannelArgumentsNativeOption>(args);
```

Comment:
Shouldn't this example live in the description of the `GrpcChannelArgumentsNativeOption` instead?